client/post-editor/post-revision.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getPostRevisions } from 'state/posts/selectors';
+import QueryPostRevisions from 'components/data/query-post-revisions';
+
+class PostRevision extends React.Component {
+	render() {
+		// FIXME: this is merely a placehodler component to test "displaying"
+		// stuff out of the redux store.
+
+		return (
+			<div>
+				<QueryPostRevisions
+					siteId={ this.props.siteId }
+					postId={ this.props.postId } />
+				<h1>{ this.props.translate( 'Post History' ) }</h1>
```

Comment:
Hi! I've found a possible matching string that has already been translated **63** times:
`translate( 'History' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).